app/controllers/notification_subscriptions_controller.rb

```diff
@@ -21,6 +21,7 @@ def create
   def confirm
     @subscriptions = NotificationSubscription.where(confirmation_token: params[:id])
     @subscriptions.each do |subscription|
+      Sendinblue::AddEmailByType.call(subscription.email, subscription.type)
```

Comment:
Taktiez (vid nizsie) toto odporucam presunut do `NotificationSubscription#confirm` a je to vybavene.